Portfolio Desktop Version

AztkMarty opened the pull request:
## THE FOLLOWING CHANGES INCLUDED IN THIS MILESTONE

-Used media queries in the CSS file to make the desktop version look like the selected template (Desktop: >=768px).
-Laid out the elements on the page using Grid and Flexbox.
-Did not implement any functionality that required JavaScript.
-Replicated the design as much as possible (font, colors, images, text, margins).
-Linters added and fixed locally.

AztkMarty committed:
Add photos and change to files html and css

Diegogagan2587 committed:
Add hover effects for .card elements, fix style sheet link

AztkMarty committed:
Add paragraph font changes

AztkMarty committed:
Merge branch 'desktopversion' of https://github.com/AztkMarty/Portfolio-Setup into desktopversion

AztkMarty committed:
Add img files

AztkMarty committed:
Add info README

AztkMarty committed:
Add README info

AztkMarty committed:
Add linters and fix issues

Alfredbis29 commented on style.css:
```diff
@@ -1,166 +1,137 @@
+  line-height: 52px;
+  margin-top: 700px;
 }
 
```
- [ ] Kindly as required, use media queries to replicate your project, using media queries is important to optimize your website for each user's viewing experience.


requirement: 
<img width="670" alt="image" src="https://user-images.githubusercontent.com/88894525/236650960-30f6bf64-18ad-4c06-9adf-d19d75b17788.png">

Alfredbis29 commented on style.css:
```diff
@@ -1,166 +1,137 @@
-  margin-bottom: 5%;
+/* --NAVIGATION MENU-- */
+
+.direction {
```
- [ ] Kindly, make your navbar look as it look in the figma template to meet the requirement:

your design:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/88894525/236651114-81ef90ff-98b1-46b4-b5d0-15d7c43c7717.png">

firgma design: 
<img width="337" alt="image" src="https://user-images.githubusercontent.com/88894525/236651089-f1209037-ddb6-429f-95f2-b8465835167a.png">

Alfredbis29 commented on style.css:
```diff
@@ -1,166 +1,137 @@
+  margin: 0;
+  width: 100%;
+}
 
```
[Optional] TO make the page look better and friendly to the user, I suggest you to add some hover, focus on buttons as shown in the figma design:  
<img width="336" alt="image" src="https://user-images.githubusercontent.com/88894525/236651225-95d037e3-1a4b-4ff0-b232-2fcd3db17371.png">

Alfredbis29 reviewed (changes requested):
#  STATUS: REQUIRED CHANGES 
 ## Hi @AztkMarty 
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ![](https://camo.githubusercontent.com/dbe642c07a12e38cdc021eb826931557cc1eb657023c3609cc10e8f3b47f06d7/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c344a7a686a56435169576765414f46612f67697068792e676966) 
 ## Highlights 
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Your project looks professional
 
 ## Required Changes 

- [ ] Kindly update the link you provided to clone your repository (`README.md` file, line `86`) to point to your actual repository.
 
<img width="371" alt="image" src="https://user-images.githubusercontent.com/88894525/236651485-cd233b09-296c-45ad-a11d-42dbe343f065.png">

- [ ]  _Others Check Under the review_


 ## Optional suggestions 

-  _Check Under the review_

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Add changes required as Nabbar, Queries, underline

AztkMarty commented:
## REQUIRED CHANGES
-Fix the navigation gaps at the top, as well as the underlining.
-Adjust top background to better fit the template.
-Improve queries to adjust to different screen resolutions from >= 768px.
-Add a link to the repository README for easy cloning.
-Improve button styling with hover effects to create a more user-friendly and visually appealing page. Ensure that the button styling aligns with the color scheme provided in the template.

binhussen commented on style.css:
```diff
@@ -1,469 +1,490 @@
-textarea {
-  height: 80px;
-  font-size: 20px;
+@media only screen and (min-width: 768px) and (max-width: 2442px) {
```
- [Optional ] Great Job for Adding the media query, however, the main usage of the media query is to have a different viewing experience, so please apply the mobile style which you have done in the previous milestone 👌

binhussen commented on style.css:
```diff
@@ -1,469 +1,490 @@
+    justify-content: center;
+  }
+
+  .header a {
+    text-decoration: none;
+  }
```
- Great Job for removing the underline, I think it would be better if you update the color also to stick the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=yfbn9fW3IN2AUzsN-0) color `#344563`  and have better UI👌

binhussen reviewed (changes requested):
Hi @AztkMarty , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Nav bar color scheme and queries fix

AztkMarty commented:
## This milestone includes the following changes:

-Fixed queries to enhance the user experience with a different viewing experience by applying the mobile style that was created in the previous milestone.
-Updated the color of the anchor tags that make up the navigation bar to match the #344563 color in Figma and improve the user interface.

AztkMarty committed:
Add fix for issue with mobile querie

AztkMarty commented:
## This milestone includes the following changes:

-Fix mobile view in media queries to ensure correct usage across multiple views.

didierganthier commented on style.css:
```diff
@@ -1,469 +1,493 @@
+  }
+}
+
+@media only screen and (min-width: 500px) and (max-width: 1800px) {
```
Well done on making some of the changes requested by the previous reviewer 💯 but some of the changes you introduced broke the design a bit, take a look 👇🏽 

![Screenshot 2023-05-07 at 00 06 09](https://user-images.githubusercontent.com/45739963/236657075-890477fd-5419-4ec7-a6c5-60f4033295c2.png)

I think the issue is related to the fact that you're using `max-width` which makes that when the screen reaches a certain `width` the media query stops applying to it. ❌

didierganthier reviewed (changes requested):
Hi @AztkMarty ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.

<img src="https://media.giphy.com/media/e56n4Z0mmkxZS/giphy.gif" />

## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- N/A

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

AztkMarty committed:
Add maximum resolution for every media querie 2560px

AztkMarty commented:
## This milestone includes the following changes:

-Add a maximum resolution of 2560 pixels to each media query to prevent breaking when viewed on high-resolution screens.

Efiamotu-1 commented on index.html:
- Unfortunately you have not addressed the issues pointed out by the previous reviewers, However i will put a detailed explanation below and hopefully it assists you in making the necessary corrections :+1: 
- [x] your design of your portfolio on the desktop version overflows the viewport and on the mobile version it is displaying the desktop view rather than the view for mobile, and i noticed this to be happening becasue you seem to have defined the same styling for both the desktop version and mobile version by using this type of media queries `@media only screen and (min-width: 375px) and (max-width: 2560px) {}`
- [x] It is important that you note that when styling with media queries and you define a min-width  this automatically implies that any change you make within that min width would start applying from that viewport so if i define a minwidth of `768px` all the stylings under this would only start applying to tablet devices, ipads and above and any styling outside of this would apply to devices below `768px` such as mobile phone. what this means is that you only need to define styling for min-width of 768px and you would be able to adapt your design to both mobile and desktop devices and to be specific you can add other min-widths. please i would like you to re-read what i just wrote and iif you need more clarifications you can reach me on slack or via this pr thread and i will respond immediately. 
- [x] It is important i explained it that way and maybe it will assist you in correcting your design structure :sparkles: :+1: 
- desktop view
![overflows](https://user-images.githubusercontent.com/89055502/236658143-1a017ce5-9a66-4877-b3cf-54f42cc83769.png)
- mobile view
![uneven](https://user-images.githubusercontent.com/89055502/236658152-6255dc63-6e48-4456-ad06-fe66b4fb9adc.png)

Efiamotu-1 reviewed (changes requested):
Hi @AztkMarty , 

![](https://camo.githubusercontent.com/a06454929d29b512682cd8b375641b59076b9e5fe1a367808e685e5bcb818de3/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f76312e59326c6b505463354d4749334e6a45784e32466d5957466c5a6a45794d7a49314f4455784d5455775a44686b595459355932526c4e6a4d334e7a4e6c4f574530596a4a684d695a6c634431324d563970626e526c636d35686246396e61575a7a583264705a6b6c6b4a6d4e305057632f7a3370696f6b77663057504838314d4f68752f67697068792e676966)

Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AztkMarty committed:
Add changes to media queries

AztkMarty committed:
Add linters errors &  media queries fix

AztkMarty commented:
## This milestone includes the following changes:
-Adjustments in the media queries. To ensure that each section is displayed correctly based on the screen size.

AztkMarty committed:
Add Flex fix

AztkMarty committed:
Add fix in linters

Shedrack-Sunday commented on index.html:
```diff
@@ -77,183 +84,254 @@ <h2>My Recent<br>Works</h2>
+                <div class="about-box about">
+                    <h2>About me</h2>
+                </div>
+            <p class="textaboutme-desktop ">
+                
+                I can help you build a product , feature or website Look through some of my work and<br>experience! If you like what you see and have a project you need coded,<br>
+                don’t hesitate to contact me.
```
- [x] Geat job you did here 👏 .. However, kindly use the right font and color as specified on the figma design.
- On Figma, the color i `background: #42526E;` and its not bold

## Figma:
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/95b9e176-e7fc-42ef-a400-aa6d89789966)

## Your work:
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/d758350b-ae99-457d-812f-061a92751833)

Shedrack-Sunday commented on index.html:
```diff
@@ -4,71 +4,78 @@
-            <!-- Support section -->
-            <p class="frame2">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don't hesitate to contact me.
+<!-- TOP SECTION -->
+    <section class="info-desktop" id="head-line">
+        <div class="info frame">
+            <h1 id="title">Hey there.<br class="jump">I’m Martin.</h1>
+            <h2 id="title2">I’m a software<br class="jump"> developer</h2>
+            <p class="frame2">I can help you build a product, feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br> 
+                    don't hesitate to contact me.
```
- [x] You refactoring on desktop should still maintain the mobile design you have. This is the whole idea behind a responsive web app, to be flexible and have a nice UI across differenet screen sizes."
- Kindly adjust the structure of this `header` section to replicate the figma design at the mobile view of `375`.  Major chnages happen to be the background image, and the hanburger button which needs to be righly placed. 
- Using the right `CSS` selectors sech as flecbox on this view port will fix this 👍 

## Your view:
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/5d126c3a-48ad-4ebd-b80a-e2c226ee64d0)

## Figma 
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/db32c9ab-403e-4101-8ca8-50f7c1977116)

Shedrack-Sunday reviewed (changes requested):
Hello @AztkMarty         👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)

## Highlights   
- Good use of GitHub features ✔️   
- Right linter check setup ✔️  

  
# Required Changes  🌵 

- [x] The major changes needed has to do with your styling which needs adjustment on the mobile view point of `375`, and a font colour on the about 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

AztkMarty committed:
Add fix background and humburger

AztkMarty committed:
Add linters errors

AztkMarty commented:
## This milestone includes the following changes:
-I put a lot of effort into fixing the design for the 375px mobile resolution.

koachcode commented on index.html:
```diff
@@ -4,33 +4,42 @@
-    <!-- Header section -->
-    <header class="menu-line">
+<!-- BACKGROUND & NAVIGATION MENU -->
+<div class="wrapper">
```
- Hi @AztkMarty The desktop design looks good, however, can you please place the decorative background images in their appropriate location as it is on the Figma design, Currently it overlaps the hero section test which makes it don't visible to read. 

- **Your Design**

![image](https://github.com/AztkMarty/Portfolio-Setup/assets/58771507/ec77e419-dac3-4fa4-b746-f02db9dc59c3)

-  **Fima Design**

![image](https://github.com/AztkMarty/Portfolio-Setup/assets/58771507/48918ff2-8d34-4f66-a8ad-43b691b0545b)

koachcode reviewed (changes requested):
Hi @AztkMarty,  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![keep-trying-youre-almost-there](https://user-images.githubusercontent.com/67087664/169483884-c624db70-793d-42be-8ebe-afef6a1cd5c3.gif)

 ## Highlights
- No linter errors :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
